docs/datascientist_onboard.md

```diff
@@ -0,0 +1,230 @@
+# Getting Started with PAI
+
+## Overview
+- OpenPAI is an open source platform that provides complete AI model training and resource management capabilities. 
+- It supports multiple types of jobs including Deep Learning, Machine Learning, Big Data, etc. 
+- It runs jobs in Docker containers and provides container schedule mechanism and user-friendly monitor system.
+- It use distributed file system HDFS to store and manage data.
+- For Deep Learning jobs, OpenPAI supports all the DL frameworks (including CNTK, TensorFlow, etc.) as long as you install the corresponding packages in your customized job container.  
+- Users need to prepare a config file in JSON format and submit it for a job submission 
+
+
+## Prerequisites
+This document assumes:
+- The PAI system has already been deployed properly and a docker registry is available to store docker images. 
+- You have done the debug of your code on your local machine to ensure it works well.
+
+
+## Basic Workflow 
+
+To run a job on PAI, you should follow the follow the following basic workflow:
+1. Prepare your data and code, upload them to HDFS.
+2. Prepare your job container
+3. Write a job JSON configuration file
+4. Register an account, then submit job
+5. Debug job
+	
+	
+## Quick start
+This part will guide you getting started  from running our example jobs  by submitting the job's json configuration file straightly without any modification. Thus you don't need to prepare the code, data and job container, which means you just need to go with 4, 5 of Basic Workflow.
+
+- [Quick start with submitting a CIFAR-10 job](https://github.com/Microsoft/pai/blob/master/examples/README.md#quickstart)
+- [Learn more example jobs](https://github.com/Microsoft/pai/tree/master/examples#offtheshelf)
+   
```

Comment:
If the CIFAR-10 example could be run end-to-end and break down step-by-step, rather than a ready-made json, it would be more understandable for me. For people who have no domain knowledge about PAI and docker(i.e. me when check in ), even I submit such an example, I don't know what happened and how could I run my job on PAI. During my DRI, some user asks me the similar question.